interface_frontend.py

```diff
@@ -114,6 +114,39 @@ def get_sprite(img_path, x=0, y=0):
     'move3': 'MOVE 3',
 }
 
+priority_labels = {}
+
+def get_priority_label(text, x, y):
```

Comment:
Napadá mě, že by se tyto dvě funkce mohly v pohodě sloučit do jedné, jen volat jednou pro priority, a jednou pro name. Stejně tak font a kotva by se mohly předat argumentem. Pak zde chybí docstringy.

Reply:
super nápad, dodělám, pak dám vědět :)

Reply:
@befeleme  hotovo tak mrkni jestli to tak může být :-)